certificate-service/domain/src/main/java/se/inera/intyg/certificateservice/domain/certificatemodel/model/FK7211CertificateSummaryProvider.java

```diff
@@ -0,0 +1,42 @@
+package se.inera.intyg.certificateservice.domain.certificatemodel.model;
+
+import java.time.format.DateTimeFormatter;
+import se.inera.intyg.certificateservice.domain.certificate.model.Certificate;
+import se.inera.intyg.certificateservice.domain.certificate.model.ElementValueDate;
+
+public class FK7211CertificateSummaryProvider implements CertificateSummaryProvider {
```

Comment:
Move this implementation to infrastructure. It should sit together with the FK7211-factory class.